src/JBrowse/Store/RemoteBinaryFile.js

```diff
@@ -233,7 +233,7 @@ return declare( null,
             callback( response, null, {nocache: nocache } );
         };
 
-        req.onreadystatechange = dojo.hitch( this, function() {
+        req.onreadystatechange = dojo.hitch( this, async function() {
```

Comment:
i do not thing that means what you think it means

Reply:
Wool meant to remove that

Reply:
Woops meant to remove that

Reply:
Can you do that for me

Reply:
Nevermind I got it :)